lib/xmerl/src/xmerl.erl

```diff
@@ -53,6 +52,10 @@ export(Content,Callback)
 
 Equivalent to [export(Data, Callback, [])](`export/3`).
 """.
+-spec export(Content, Callback) -> ExportedFormat when
```

Comment:
[Maybe my xmerl ignorance shines through] What is `Callback`? Function name?
Maybe use a type with a name that explains this:

Reply:
Let me leave this comment to discuss in some future. :-)